wordpress-mobile/WordPress-iOS
# 1. Showing connection error alert in SharingViewController

Fixes: #11002 
 Other unfriendly messages were updated in: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/83

Previously if the call for syncPublicizeServices failed due to no connectivity the error message would show "Publicize service synchronization failed" which does not provide information on why they failed or the state of connectivity.

Added reachability to the view controller and showing a connection error instead of the failure error in case network is not reachable.
Old
![img_3154](https://user-images.githubusercontent.com/1335657/52601384-72fdc200-2e13-11e9-876f-8dc920ed9a35.PNG)

New
![img_3155](https://user-images.githubusercontent.com/1335657/52601377-6b3e1d80-2e13-11e9-9a0a-6c9b7addec56.PNG)


To test:
1.
- Go to my site
- Go to Sharing
- Turn offline 
- Notice the "No internet connection alert"

2. 
- Be offline
- Go to my site
- Go to sharing
-  Notice the "No internet connection alert"



Update release notes:
